package.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "snips-react",
+    "displayName": "snips-react",
+    "version": "0.0.1",
+    "description": "A snippet pack to make you more productive working with React",
+    "publisher": "causecode",
+    "author": "CauseCode Technologies",
+    "license": "MIT",
+    "engines": {
+        "vscode": "^1.19.0"
```

Comment:
Is caret really needed?

Reply:
Yes, it indicates compatibility with VS Code version `1.19.0` and onwards and cannot be `*`. More details [here](https://code.visualstudio.com/docs/extensionAPI/extension-manifest).